+ added method 'paginate' for relationships and method 'chunk' for query builder

LookinGit opened the pull request.

RomainLanz commented:
Hey @LookinGit! 👋 

Thanks for your contribution.
May you add tests to test this feature?

thetutlage commented:
Also the method added is `chunk` and we have `paginate` in the title.

Can u please take a minute to explain, what exactly the feature is supposed to do as per you.

LookinGit commented:
Hey, @RomainLanz and @thetutlage 
Sorry, at first it was PR with method `paginate`, but then I added PR with method `chunk` in the same branch. I renamed PR.
Should I create two new PR instead of this?

thetutlage commented:
Hello @LookinGit 

I actually removed `chunk` method for a reason, since we can use stream instead of buffering the data in memory http://knexjs.org/#Interfaces-stream

LookinGit commented:
@thetutlage I changed method `chunk` using streams

LookinGit commented:
Created new PR only with `paginate` method, without `chunk`
[https://github.com/adonisjs/adonis-lucid/pull/222](url)
